.github/workflows/refresh-doc.yml

```
@@ -0,0 +1,75 @@
+name: refresh-doc
+on:
+  push:
+    branches:
+      - hannah-github-actions
```

I guess this should be changed to `master` once ready for merging?

```suggestion
      - master
```

---

Eventually yes, but atm I don't see a huge reason to change this to master, since we don't have the live web server working anyways. I think we can still do all the testing on this branch and make little changes in the markdown files for testing before relying on the master branch.